Kickstarter-iOS/Features/LoginTout/Controller/LoginToutViewController.swift

```diff
@@ -187,12 +187,29 @@ public final class LoginToutViewController: UIViewController, MFMailComposeViewC
         self?.pushSignupViewController()
       }
 
-    self.viewModel.outputs.logIntoEnvironment
+    self.viewModel.outputs.logIntoEnvironmentWithApple
       .observeValues { [weak self] accessTokenEnv in
         AppEnvironment.login(accessTokenEnv)
         self?.viewModel.inputs.environmentLoggedIn()
       }
 
+    self.viewModel.outputs.logIntoEnvironmentWithFacebook
+      .observeValues { [weak self] accessTokenEnv in
+        guard let _self = self else { return }
```

Comment:
Can you not use `_self` just because our conventions don't typically have this.
We usually use `self` or `strongSelf` (preferred).

Reply:
sure thing. The ViewController I was referencing, `ChangePasswordViewController`, uses `_self`. I'll update this to use the more preferred convention.

Reply:
Oh really, oh I haven't seen it before thanks for changing it to `strongSelf`, let's both use it going forward.